packages/wonder-blocks-timing/src/hooks/internal/use-updating-ref.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import {useEffect, useRef} from "react";
+
+/**
+ * Returns a ref whose .current value is updated whenever
+ * the `value` passed to this hook changes.
```

Comment:
maybe add something about "this is great for values that you want to reference from within a useCallback or useEffect event listener, without re-triggering the effect when the value changes"?

Reply:
Great suggestion!  I gotta up my docs game and go beyond just what a thing does to why you'd want to do such a thing.